boards/xtensa/esp32/esp32-devkitc/src/esp32_reset.c

```diff
@@ -24,11 +24,20 @@
 
 #include <nuttx/config.h>
 
+#include <stdlib.h>
+#include <debug.h>
+#include <assert.h>
 #include <nuttx/arch.h>
 #include <nuttx/board.h>
 
+#include "esp32_systemreset.h"
+
 #ifdef CONFIG_BOARDCTL_RESET
 
+#if CONFIG_BOARD_ASSERT_RESET_VALUE == EXIT_SUCCESS
+#  error "CONFIG_BOARD_ASSERT_RESET_VALUE is equal to EXIT_SUCCESS"
```

Comment:
```suggestion
#  error "CONFIG_BOARD_ASSERT_RESET_VALUE must not be equal to EXIT_SUCCESS"
```
Suggestion for making the error message clearly state the reason for failure.

Reply:
Done, thanks